Makefile

```diff
@@ -41,7 +41,9 @@ clean:
 	@echo "Cleaning"
 	@rm -fr apiary.apib
 
-publish: apiary.apib
+publish: bundler apiary.apib publishOnly
```

Comment:
This is target will have timing issues due to these three dependencies all from publish. They can run at the same time and thus publishOnly can run before bundler in completed. Since the publishOnly target does really depend on bundler in can fail.